cumulus_library/__init__.py

```diff
@@ -0,0 +1,5 @@
+from importlib import metadata
+
+__version__ = metadata.version(__package__)
```

Comment:
Taste may vary, but I've found it more convenient to just define `__version__ = "0.3.2"` here in this file and set `dynamic = ["version"]` in pyproject.toml, and it will then read it from this file. (see ctakesclient or cumulus-etl for examples).

Reply:
ok, i came around on this.